knora/dsp_tools.py

```diff
@@ -142,31 +140,27 @@ def program(user_args: list[str]) -> None:
         exit(0)
 
     if args.action == 'create':
-        if args.lists:
-            if args.validate:
+        if args.lists_only:
+            if args.validate_only:
                 validate_list_with_schema(args.datamodelfile)
             else:
                 create_lists(input_file=args.datamodelfile,
-                             lists_file=args.listfile,
                              server=args.server,
                              user=args.user,
                              password=args.password,
-                             verbose=args.verbose,
                              dump=args.dump)
         else:
-            if args.validate:
-                if validate_ontology(args.datamodelfile):
+            if args.validate_only:
+                if validate_project(args.datamodelfile):
+                    print('Data model is syntactically correct and passed validation.')
                     exit(0)
-                else:
-                    exit(1)
```

Comment:
The behaviour of `validate_project()` changed: It returns True or raises an Error with detailed explanation. So, it's not necessary any more to `exit()`.